tests/e2e/module_templates/moduletemplate_template_operator_regular.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: operator.kyma-project.io/v1beta2
```

Comment:
I prepending this file with a descriptive comment that describes the resource in it and how this resource is used in the test suite.

Reply:
that's the ModuleTemplate that got generated from the create Module command, I don't think we should alter it